Extensions/Effects/godray-pixi-filter.js

```diff
@@ -1,12 +1,16 @@
 gdjs.PixiFiltersTools.registerFilterCreator('Godray', {
   makePIXIFilter: function(layer, effectData) {
     var godrayFilter = new PIXI.filters.GodrayFilter();
-
+    godrayFilter._animationTimer = 0;
     return godrayFilter;
   },
   update: function(filter, layer) {
-    if (filter.animated) {
-      filter.time += layer.getElapsedTime() / 1000;
+    if (filter.animationFrequency !== 0) { 
+      filter._animationTimer += layer.getElapsedTime() / 1000;
+      if (filter._animationTimer >= 1 / filter.animationFrequency) {
+        filter.time += layer.getElapsedTime() / 1000;
```

Comment:
Same here, time must be updated continuously. 
Revert the changes and consider using animationSpeed for this.